docs/src/pages/[platform]/connected-components/liveness/react-props.ts

```diff
@@ -78,13 +78,13 @@ export const FACE_LIVENESS_DETECTOR_CONFIG = [
   {
     name: `binaryPath?`,
     description:
-      'Overrides the WASM backend binary CDN path, the default is https://cdn.jsdelivr.net/npm/@tensorflow/tfjs-backend-wasm@3.11.0/dist/. When overriding this path ensure that the wasm version matches the version of @tensorflow/tfjs-backend-wasm installed by npm.',
+      'Overrides the WASM binary path, the default is https://cdn.liveness.rekognition.amazonaws.com/face-detection/tensorflow/tfjs-backend-wasm/3.11.0/. When overriding this path ensure that the wasm version matches the version of @tensorflow/tfjs-backend-wasm installed by npm.',
     type: `string`,
   },
   {
     name: `faceModelUrl?`,
     description:
-      'Overrides the Blazeface model and weights bin CDN URL default is https://tfhub.dev/tensorflow/tfjs-model/blazeface/1/default/1/model.json?tfjs-format=file.',
+      'Overrides the Blazeface model and weights bin CDN URL default is https://cdn.liveness.rekognition.amazonaws.com/face-detection/tensorflow-models/blazeface/0.0.7/model/model.json.',
```

Comment:
Is it possible to do wrap the CDN URL in backticks to make it look like code?

```suggestion
      'Overrides the Blazeface model and weights bin CDN URL default is https://cdn.liveness.rekognition.amazonaws.com/face-detection/tensorflow-models/blazeface/0.0.7/model/model.json.',
```

Reply:
This unfortunately does not seem to work, it just shows the back ticks 
<img width="1205" alt="Screenshot 2023-07-12 at 2 56 36 PM" src="https://github.com/aws-amplify/amplify-ui/assets/68032955/ce5410a1-7aa0-41e1-8ea5-1874e8f8a5ca">